cylc/uiserver/resolvers.py

```diff
@@ -51,7 +51,7 @@ class InvalidSchemaOptionError(CylcError):
 CLEAN = 'clean'
 OPT_CONVERTERS: Dict[str, Dict[str, Union[Callable, None]]] = {
     CLEAN: {
-        'rm': lambda opt, value: ('rm_dirs', [value]),
+        'rm': lambda opt, value: ('rm_dirs', [value] if value else None),
```

Comment:
This resulted in `['']` which is "Truthy" effectively disabling the clean operation.